understanding/22/accessible-authentication.html

```diff
@@ -86,6 +86,8 @@ <h2>Intent of Accessible Authentication</h2>
         <h3>Copy and paste</h3>
         <p>Copy and paste can be relied on to avoid transcription. Users can copy their login credentials from a local source (such as a standalone third-party password manager) and paste it into the username and password fields on a login form, or into a web-based command line interfaces asking for a password. Blocking people from pasting into authentication fields, or using a different format between the copied text and the input field (for example, "Enter the 3rd, 4th, and 6th character of your password"), would force the user to transcribe information and therefore fail this criterion, unless another method is available.</p>
 
+        <p>If the copy and paste requires the user to have and use multiple devices (e.g., copy code from phone, email to yourself, open on laptop) copying multiple times, then it would still be considered a Cognitive Function Test. Some systems do allow for simple copy and paste across devices (e.g., tap code on phone, paste on laptop), which do not involve copying multiple times, but you would need all users to have easy access to that for it to be considered <a href="#dfn-accessibility-supported">accessibility supported</a>.</p>
```

Comment:
```suggestion
        <p>If the copy and paste requires the user to have and use multiple devices (e.g., copy code from phone, email to yourself, open on laptop) copying multiple times, then it would still be considered a Cognitive Function Test and fail this criterion. Some systems do allow for simple copy and paste across devices (e.g., tap code on phone, paste on laptop), which do not involve copying multiple times, but you would need all users to have easy access to that for it to be considered <a href="#dfn-accessibility-supported">accessibility supported</a>.</p>
```